docs/source/conf.py

```diff
@@ -141,15 +141,18 @@
     'dark_css_variables': {'admonition-title-font-size': '0.95rem',
                            'admonition-font-size': '0.92rem'},
     'light_css_variables': {'admonition-title-font-size': '0.95rem',
-                            'admonition-font-size': '0.92rem'}
+                            'admonition-font-size': '0.92rem'},
+    "announcement": 'PTB has undergone significant changes in v14. Please read the documentation '
+                    'carefully and also check out the transition guide in the '
+                    '<a href="#TODO: Add link">wiki</a>',
```

Comment:
```suggestion
                    '<a href="https://github.com/python-telegram-bot/python-telegram-bot/wiki">wiki</a>',
```

should be enough for now and is a better fallback in case we forget adding the actual link in time :)